templates/new/batch_runner.py.txt

```diff
@@ -0,0 +1,23 @@
+from surround import Runner
+from stages import {project_name}Data
+
+
+class BatchRunner(Runner):
+    def prepare_runner(self):
+        # Load data from files or directory here
+        self.raw_data = "TODO: Load raw data here"
+        self.data = {project_name}Data()
+        self.assembler.run_on_batch_mode()
+        self.assembler.init_assembler(self.data)
+
+    def prepare_data(self):
```

Comment:
Why do we need this method? Can we just pass in the `input_data` into the `run` command on line 21?